flocker/node/agents/test/test_blockdevice.py

```diff
@@ -126,10 +119,21 @@
 from ....common.test.test_thread import NonThreadPool, NonReactor
 from ....common import RACKSPACE_MINIMUM_VOLUME_SIZE
 
-CLEANUP_RETRY_LIMIT = 10
-LOOPBACK_ALLOCATION_UNIT = int(MiB(1).to_Byte().value)
-# Enough space for the ext4 journal:
-LOOPBACK_MINIMUM_ALLOCATABLE_SIZE = int(MiB(16).to_Byte().value)
+from ..testtools import (
+    FakeCloudAPI,
+    detach_destroy_volumes,
+    fakeprofiledloopbackblockdeviceapi_for_test,
+    loopbackblockdeviceapi_for_test,
+    # NB: Think carefully before moving this.
+    # Backend driver docs used to instruct developers to import this from here.
```

Comment:
:question: It'd be nice to have instructions here for if/when this could be removed.

Suggest adding a link to https://github.com/search?utf8=%E2%9C%93&q=make_iblockdeviceapi_tests+extension%3Apy&type=Code&ref=searchresults and saying that until that returns no drivers importing from this package, it must be included here.

or a link to a FLOC issue that links to that search.